docs/semgrep-code/notifications.md

```diff
@@ -56,15 +56,149 @@ To view, add, remove, disable, or enable your saved channels:
 2. In the **[Rule board](https://semgrep.dev/orgs/-/board)**, click the <i class="fa-solid fa-gear"></i> **gear** icon to enable or disable an integration.
     ![Screenshot of Semgrep's Rule board integration modal](/img/rule-board-integrations.png)
 
-## Adding notification channels
+## Setting up and customizing notification channels
 
 ### Slack
 
 <!-- When referring to the Slack integration, let's consistently call it the "Semgrep Slack app" (app in lower case). This is consistent with the terminology used by Slack (they refer to these integrations as apps in their product pages. It also makes sure that the user knows what "app" we're referring to.
 
 Prefer "Semgrep Slack app" over "Semgrep Slack integration" or "Slack integration" or any other variant. -->
 
-<ProcedureIntegrateSlack />
+<!-- <ProcedureIntegrateSlack /> -->
+
+The Semgrep Slack app enables Semgrep Cloud Platform to notify you of new findings after every scan. By receiving notifications within your Slack workspace, developers and security engineers are able to see findings without leaving their current environment. This can lessen the friction between detecting a finding, triaging it, and resolving it.
+
+You can select which channels in your Slack workspace will receive findings. You can also choose to receive findings only for certain repositories (projects) or certain **policies**. 
+
+**Policies **typically correspond to the different Rule board columns. For example, you can choose to receive notifications only for findings generated by rules from the Rule Board's Blocking column.
+
+:::info PREREQUISITE
```

Comment:
Optional suggestion: These headers are all rendered as capitalized, this is just source code consistency edit (feel free to dismiss). 
```suggestion
:::info Prerequisite
```